app/models/maestrano/connector/rails/concerns/entity.rb

```diff
@@ -465,7 +465,7 @@ def consolidate_and_map_singleton(connec_entities, external_entities)
     # proc is a lambda to create a batch_op from an element of the array
     # Perform batch calls on Connec API and parse the response
     def batch_calls(array_with_idmap, proc, connec_entity_name, id_update_only = false)
-      request_per_call = @opts[:request_per_batch_call] || 100
+      request_per_call = @opts[:request_per_batch_call] || 50
```

Comment:
can we move this to a constant at the start of the file like BATCH_SIZE=50

Reply:
Good point but I'm actually refactoring the whole paging and batch logic in another branch so that's  going to disappear.

Reply:
In that case, Everything looks good to me.

Reply:
👍